### PR TITLE
Making it easier for people to know they need Entrez Direct.

### PR DESCRIPTION
**Pull request by adamnovak**
I didn't have Entrez Direct installed, so I was getting a bunch of empty files and no error messages.

I fixed the script to stop if the Entrez Direct tools don't work (or anything else bad happens) and added a note about the dependency.

Also, I was only running the scripts in the first place because the Arvados dataset seems to itself be a collection of empty files today. I know that that wasn't the case before, since I peeked at the FASTAs there and they had data. Has the dataset expired or something?

### CODE REVIEW

**Comment by nouyang-curoverse** [comment]
@adamnovak  Apologies for the delayed response, I hadn't set up my github notifications correctly. Thanks for the bug report & pull request (merged the latter) and sorry about all the issues you've run into.

I easily replicated the bug, so I created new URLs and updated my Readme.md, and also filed a bug report on Arvados:
https://arvados.org/issues/5915. 

tl;dr I'm not sure why the link gives you 0 byte files, but try the new links :)

**Comment by adamnovak** [comment]
The new links seem to work. Thanks!
